tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -151,6 +151,38 @@ public function test_are_site_fields_synced() {
 		$this->assertEquals( $failure_response_data, false );
 	}
 
+	public function test_url_handles_staging() {
```

Comment:
Let's update our GHA workflow files for PHPUnit tests to install runkit7 extension. See here: https://github.com/shivammathur/setup-php#heavy_plus_sign-php-extension-support

Reply:
@eugene-manuilov does that support PHP8? If not maybe we can find a simpler way to test this 🤔

Reply:
Good point, @aaemnnosttv. The extension [description page](https://pecl.php.net/package/runkit7) says `PHP Version: PHP 7.2.0 or newer` which means that we can't use it with PHP5.6 and doesn't surely mean that it will definitely work with PHP8 :thinking: I think we should try it and if it works for GHA with PHP7+, then it's fine, tests for other versions will be just skipped.

Reply:
Turns out it works on PHP 8 https://github.com/google/site-kit-wp/pull/3495/checks?check_run_id=2821088831, for PHP 5.6 we use the old deprecated runkit extension https://github.com/google/site-kit-wp/pull/3495/checks?check_run_id=2821088699 covers all of our bases.

Reply:
Awesome! Thanks, @ivankruchkoff.